src/lib.rs

```diff
@@ -1390,6 +1392,85 @@ impl Default for StashFlags {
     }
 }
 
+macro_rules! define_attr_inspector {
```

Comment:
Could this be located in a submodule for organization, perhaps `src/attr.rs`?

(still reexported from the root, however)